GitHubA11yFixes.user.js

```diff
@@ -157,6 +114,85 @@ function onNodeAdded(target) {
 	}
 }
 
+function onNodeAdded(target) {
+	var elem;
+	if(document.location.pathname.match(/\/(?:[^\/]+\/[^\/]+\/)?notifications\/?/)){

```

Comment:
Just to clarify, I assume there is both an aggregated notifications page (github.com/notifications) and per-project notifications pages (github.com/nvaccess/axSGrease/notifications), which is what this is matching? I don't have web notifications enabled myself.

Reply:
yes